### DISCUSSION TITLE
Album cover editing for non collection songs

### DISCUSSION THREAD

**Discussion by jakubbalas** [strawberrymusicplayer/strawberry#876] [Q&A] [answered]
Hello, 

I have just a nosy question about Album cover art change feature. Currently it is only allowed for songs that are in collection.
For me and my workflow - It would be cool feature to allow it for any currently playing song but I respect the decision, just curious if it was a security or different reason. 

Please keep up the great work, I really enjoy the player ! 

Thank you !

**Comment by jonaski** [comment, marked as answer]
Because of how playlists are managed in strawberry (based on collection row ID's), it's not easy to update cover art to playlist items that are not collection specific so the covers are remembered, that's why I decided to disable the features for non-collection songs.
